spring-cloud-gcp-data-spanner/src/test/java/org/springframework/cloud/gcp/data/spanner/test/TradeRepository.java

```diff
@@ -0,0 +1,31 @@
+/*
+ *  Copyright 2018 original author or authors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.springframework.cloud.gcp.data.spanner.test;
```

Comment:
Maybe rename the package to `support` instead of `test`?

Reply:
These are specifically testing related helper classes. 
I see what you mean, but for me, `support` would suggest there is an src/main package named support and that's what we are testing.

Reply:
IMO this would go to the `domain` package as well

Reply:
how about `test.domain`?

Reply:
Fine with me